TVOAppliance/hf_upgrade.sh

```diff
@@ -76,6 +76,10 @@ function install_package()
 
 	echo "Installing $outfile for Yoga release"
 
+	echo "Before installation disable mariadb and rabbitmq-server"
```

Comment:
pls modify the test to "Disabling old and deleted MariaDB and Rabbitmq-server yum repositories"